charts/presidio/templates/recognizers-store-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{ $fullname := include "presidio.recognizersstore.fullname" . }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullname }}
+  labels:
+    app: {{ $fullname }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
```

Comment:
A single replica won't provide DR/HA within the cluster. 

Setting replicas to 2 will ensure mutliple instance exist in the cluster at any time and when combinded with [PodAntiAffinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#never-co-located-in-the-same-node) ensure that both pods don't run on a single node. 

With that in place a single node failuring or becoming stuck in the cluster won't cause a service outage.

Reply:
This is a default value, same for the others micro services.
I opened a new task to keep track and consider how we change this